packages/client/tests/memory/_utils/buildMemoryTest.ts

```diff
@@ -0,0 +1,16 @@
+import esbuild from 'esbuild'
+
+import { MemoryTestDir } from './MemoryTestDir'
+
+/**
+ * Bundles memory test file and all its dependencies with esbuild
+ * @param testDir
+ */
+export async function buildMemoryTest(testDir: MemoryTestDir): Promise<void> {
```

Comment:
We are not using `esbuild-register` (or any other on-import ways of transpiling the code) in this suite to avoid polluting the heap with artifacts of their work.